src/modules/authorizer/role/AUT_Roles_v1.sol

```diff
@@ -131,13 +131,10 @@ contract AUT_Roles_v1 is
         _setRoleAdmin(DEFAULT_ADMIN_ROLE, ORCHESTRATOR_OWNER_ROLE);
 
         // Set up MANAGER role structure:
-        // -> set OWNER as admin of DEFAULT_ADMIN_ROLE
         _setRoleAdmin(ORCHESTRATOR_MANAGER_ROLE, ORCHESTRATOR_OWNER_ROLE);
         // grant MANAGER Role to specified address
         _grantRole(ORCHESTRATOR_MANAGER_ROLE, initialManager);
-
-        // If there is no initial owner specfied or the initial owner is the same as the deployer
-
+        // grant Owner Role to initial owner address
```

Comment:
Above, the role names are capitalized (`MANAGER`, `OWNER`), here the first `Owner` is not